plugins/dfu/dfu-target.c

```diff
@@ -472,6 +474,52 @@ dfu_target_status_to_error_msg (DfuStatus status)
 	return NULL;
 }
 
+static gboolean
+dfu_target_poll_in_manifest (DfuTarget *target, GError **error)
+{
+	DfuTargetPrivate *priv = GET_PRIVATE (target);
+	DfuStatus status;
+	guint polling_count = 0;
+
+	/* get the status */
+	if (!dfu_device_refresh (priv->device, error))
+		return FALSE;
+
+	/* wait for DFU_STATE_DFU_MANIFEST to not be set */
+	while (dfu_device_get_state (priv->device)
+					== DFU_STATE_DFU_MANIFEST_SYNC ||
```

Comment:
just make a long line, 80 chars is a guideline, not a limit :)

Reply:
I will make the change. I thought the 80-chars was mandatory.